src/main/drivers/compass_hmc5883l.h

```diff
@@ -17,7 +17,7 @@
 
 #pragma once
 
-#include "drivers/io.h"
```

Comment:
Well there's lot's of header file cleanup required, so we can add the directory prefix in a subsequent PR.